### PR TITLE
Added documentation for conditional operators

### PR DESCRIPTION
**Pull request by mluisbrown**
Also:
* fixed table of contents links
* fixed a couple of other small errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +145,32 @@ The order of the expression above is:
 2. `Section() |---+ Node()` => `Section`
 3. `Box() |-+ Section()` => `Box`
 
+#### Conditional operators ❓
+
+In addition to the normal `|-+` and `|---+` concatenation operators, Bento has conditional concatenation operators:
+* `|-?` for `Section`
+* `|---?` for `Node`
+
+They are used with the static function `.iff` for
```
Comment (diegopetrucci) [inline README.md]: for?

File: README.md [hunk 2]
```diff
@@ -143,6 +145,32 @@ The order of the expression above is:
...
 
+#### Conditional operators ❓
+
+In addition to the normal `|-+` and `|---+` concatenation operators, Bento has conditional concatenation operators:
+* `|-?` for `Section`
+* `|---?` for `Node`
+
+They are used with the static function `.iff` for 
+
+They work with `Optional` and `Bool` and are used with the functions `.iff` for `Bool` and `.some` for `Optional`:
```
Comment (diegopetrucci) [inline README.md]: What about something like

```
They are used to execute a block for the `Bool` and `Optional` happy path, via the `.iff` and `.some` functions`.

File: README.md [hunk 3]
```diff
@@ -143,6 +145,32 @@ The order of the expression above is:
 2. `Section() |---+ Node()` => `Section`
 3. `Box() |-+ Section()` => `Box`
 
+#### Conditional operators ❓
+
+In addition to the normal `|-+` and `|---+` concatenation operators, Bento has conditional concatenation operators:
```
[the remark this answers is not in the document]
Comment (mluisbrown) [reply README.md]: normal as opposed to conditional. Maybe just get rid of the word "normal" altogether?

File: README.md (hunk 1 above)
Comment (mluisbrown) [reply README.md]: 🤦‍♂️

**Pull request merged**